base/docs/basedocs.jl

```diff
@@ -600,6 +600,16 @@ catch
     @warn "Could not write file."
 end
 ```
```

Comment:
Can you add blank lines before and after the code blocks?

Reply:
yes